# HG changeset patch
# User Ernst Reissner <rei3ner@arcor.de>
# Date 1487387092 -3600
#      Sat Feb 18 04:04:52 2017 +0100
# Branch stable
# Node ID 895b8edb4f3c03a54e58411258ee76684d8b9358
# Parent  b93e1d29790e6cd70a5a52100111d3b48368aedb
doc: Fix build of docs broken in sub2ind (bug #50348).

* sub2ind.cc (Fsub2ind): Delete accidental '%' at end of @example.

diff --git a/libinterp/corefcn/sub2ind.cc b/libinterp/corefcn/sub2ind.cc
--- a/libinterp/corefcn/sub2ind.cc
+++ b/libinterp/corefcn/sub2ind.cc
@@ -232,17 +232,17 @@ ind = [2, 8];
 @end example
 
 If the number of output subscripts exceeds the number of dimensions, the
 exceeded dimensions are set to @code{1}.  On the other hand, if fewer
 subscripts than dimensions are provided, the exceeding dimensions are merged
 into the final requested dimension.  For clarity, consider the following
 examples:
 
-@example%
+@example
 @group
 ind  = [2, 8];
 dims = [3, 3];
 ## same as dims = [3, 3, 1]
 [r, c, s] = ind2sub (dims, ind)
     @result{} r =  2   2
     @result{} c =  1   3
     @result{} s =  1   1
